src/sagas/index.js

```diff
@@ -53,6 +53,15 @@ const changeRecordSinglesToArrays = function(record) {
             if(record.data.attributes.prefix){
                 record.data.attributes.prefix = String(record.data.attributes.prefix._text);
             }
+            if(record.data.attributes.identifiers){
+                if(record.data.attributes.identifiers[0]){
```

Comment:
Hi @eddiesarevalo There is a list of identifiers so that might not happen on the first one.

I am also worrying about the fact that that might happen somewhere else in the XML. Would there be a solution on parameterizing the conversion so that we never have there _text attributes ?

Thanks

Reply:
Hello @tloubrieu-jpl You are right about the list. This will only do the first one. I need to set it to do a list.

The main problem I faced when dealing with the conversion between xml and json was coming up with a way to remove the _text attributes and set the arrays for values.

I'm using this library: https://www.npmjs.com/package/xml-js

It converts xml to json and back, however there are some limitations to what any converter can do. The issues I faced here were first, that the converter does not know the types of what a json object should be from just the xml. It only sees text. 

For example the xml tag `<publicationYear>2020</publicationYear>` is it text or number? The converter will just detect it as a string and put it as `{_text:'2020'}`

Second the converter doesn't know if an item in xml is an array or single item.

Here is the example of the problem. Given the tag `creators`
```html
<creators>
     <nameType>Personal</nameType>
     <name>Arevalo Eddie</name>
</creators>
```

The converter doesn't know this is not a single object:
```javascript
creators:{
          nameType:'Personal'
          name:'Eddie'
}
```

and should instead be an array:
```javascript
creators:[{
          nameType:'Personal'
          name:'Eddie'
}]
```

I created that section of code above to do the conversions that the converter would not be able to do. First to remove the _text if it exists and second to convert to arrays the objects that should be an array.